kolibri/plugins/learn/assets/src/my_downloads/views/DownloadsList/index.vue

```diff
@@ -83,10 +86,13 @@
                   :color="download.status === 'PENDING' ? $themeTokens.annotation : null"
                   class="icon"
                 />
-                <KCircularLoader
-                  v-if="download.status === 'IN_PROGRESS'"
-                  class="icon"
-                />
+                <div v-if="download.status === 'IN_PROGRESS'" class="inline-loader">
+                  <KCircularLoader
+                    :size="20"
+                    disableDefaultTransition="true"
```

Comment:
The change needs to be made here as well. Once that is done, this PR should be ready to merge :)

Reply:
Apologies, I was in a bit of a hurry yesterday and missed it. Thanks @LianaHarris360  for keeping an eye out! :clinking_glasses: